BigBaseV2/src/views/vehicle/view_spawn.cpp

```diff
@@ -24,66 +15,71 @@ namespace big
 		ImGui::SameLine();
 		ImGui::Checkbox("Spawn Maxed", &g->spawn.spawn_maxed);
 
-		components::input_text_with_hint("Model Name", "Search", model, sizeof(model), ImGuiInputTextFlags_EnterReturnsTrue, []
-		{
-			const auto ped = self::ped;
+		static char plate[9] = { 0 };
+		strncpy(plate, g->spawn.plate.c_str(), 9);
 
-			const auto location = ENTITY::GET_OFFSET_FROM_ENTITY_IN_WORLD_COORDS(ped, 2.f, 2.f, 0.f);
-			const auto veh = vehicle::spawn(model, location, g_local_player->m_player_info->m_ped->m_navigation->m_right.x + 90.f);
+		ImGui::SetNextItemWidth(300.f);
+		if (ImGui::InputTextWithHint("Plate", "Plate Number", plate, sizeof(plate))) {
```

Comment:
Change this to our text input component, as it currently doesn't disable your keys in game.